AK/HashTable.h

```diff
@@ -143,6 +144,14 @@ class HashTable {
     HashTable() = default;
     explicit HashTable(size_t capacity) { rehash(capacity); }
 
+    HashTable(std::initializer_list<T> list)
+    requires(!IsLvalueReference<T>)
+    {
+        ensure_capacity(list.size());
```

Comment:
Not a huge fan of this, as it encourages a pattern where we have no way to handle OOM.
Maybe instead we could take an initializer list as a factory method on `HashTable` and return an error if we fail to allocate? 

If you end up keeping this, please atleast `#ifdef` this so it's not available when compiling for the Kernel.

Reply:
Hmm that is a good point, I'll have a look how to improve this

Reply:
Changed it to a failable `of` method (name can be bikeshed, this is something from java)